### ISSUE TITLE
Don't allow user to run scancode if package not installed

### ISSUE THREAD

**Issue by GMishx** [fossology/fossology#2202]
### Description

After installing FOSSology, if the system does not have [scancode-toolkit](https://github.com/nexB/scancode-toolkit) installed, the option to schedule the agent is shown to the user.

The UI should check if the tool is installed before showing the option to user.

#### Steps followed and expected result

Check if `scancode` executable is installed before showing the option to schedule the agent by user.

#### Logs

```
2022-04-04 05:22:49 scancode [0] :: JOB[12].scancode[27670.localhost]: "sh: 1: scancode: not found"
2022-04-04 05:22:49 scancode [0] :: JOB[12].scancode[27670.localhost]: "sh: 1: scancode: not found"
2022-04-04 05:22:49 scancode [0] :: JOB[12].scancode[27670.localhost]: "FATAL scancode_wrapper.cc.99: could not execute scancode command: scancode -lceu --custom-output - --custom-template scancode_template.html /srv/fossology/repository/localhost/files/cc/9f/01/cc9f01f2b66d95389d573888a017ac0580610510.21cb8c0b9ba0230cfddec93a3e925c62.309406 --quiet  --license-text --license-score 50 "
2022-04-04 05:22:49 scancode [0] :: JOB[12].scancode[27670.localhost]: agent failed with error code 1
2022-04-04 05:22:49 scancode [0] :: JOB[12].scancode[27670.localhost]: agent failed, code: 1
```

Related to #2171

**Comment by GMishx** [comment]
There can be a check if scancode is installed and then only register the plugin at
https://github.com/fossology/fossology/blob/77a85ca0396a7545939656a8f9b8bf9b2620cc39/src/scancode/ui/agent-scancode.php#L148

**Comment by kannan1997** [comment]
Hi @GMishx , 

Can we add the below line in Dockerfile (to install scancode-toolkit via Dockerfile)?

```
# For using scancode-toolkit

RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1 \
 && apt-get update \
 && python -m pip install --upgrade pip setuptools wheel \
 && apt-get install python3.7-dev \ 
 && python -m pip install scancode-toolkit

``` 
 This is tested internally and working as expected internally. 
 
 **How to test?**
 
 Run the below command inside the container. 

``` 
root@host:/fossology# scancode --version
/usr/local/lib/python3.7/dist-packages/cluecode/copyrights.py:3388: FutureWarning: Possible set difference at position 3
  re.MULTILINE | re.UNICODE
ScanCode version 30.1.0
ScanCode Output Format version 1.0.0
SPDX License list version 3.14

```

**Comment by GMishx** [comment]
Fixed by f431c98c7610c0ae1dc0cab0d3359b5d86f84d5b

**Comment by GMishx** [comment]
Hi @kannan1997 , the installation of scancode was fixed in https://github.com/fossology/fossology/pull/2221

We moved it to Python installation pipeline from C/C++ installation pipeline.
